### DISCUSSION TITLE
Summer Sale evend & ClaimItem

### DISCUSSION THREAD

**Discussion by botka4aet** [chr233/ASFEnhance#270] [提问求助 / Support]
Event started and you can claim 1 item per day
Buy !claimitem doesn't work

`There is no item to claim`

ASF 6.0.3.4
ASFEnhance 2.1.8.3

Bug?

**Comment by WaTaShi-Rami** [comment]
i think what you mean is the discovery que, where you get 1 card drop, that command should be:  EXPLORER 

im not sure if it still works but i tested it and it worked just takes a while.

**Reply by krakyz** [reply]
EX doesn't work for me for some reason... Says it'd start immediately, but nothing happens :(

**Reply by WaTaShi-Rami** [reply]
yes, after that it takes some time to start, steam is a bit laggy but it worked for me

**Reply by krakyz** [reply]
yeah, can see it doesn't even load the wishlist

**Reply by insidioso** [reply]
> yes, after that it takes some time to start, steam is a bit laggy but it worked for me

did you solve the problem? ive got the same... plugin doesnt work for me

**Reply by Shezik1** [reply]
> > yes, after that it takes some time to start, steam is a bit laggy but it worked for me
> 
> did you solve the problem? ive got the same... plugin doesnt work for me


I have the same problem

**Reply by chr233** [reply]
> i think what you mean is the discovery que, where you get 1 card drop, that command should be: EXPLORER
> 
> im not sure if it still works but i tested it and it worked just takes a while.

ASF change its bot config format, `AutoSteamSaleEvent` now merge in a option of `FarmingPerferences`, check if you enabled this feature,

ASFEnhance now supported "ENABLEAUTOSTEAMSALEEVENT` command, use this command to enable this feature for speect bots

**Reply by chr233** [reply]
> > > yes, after that it takes some time to start, steam is a bit laggy but it worked for me
> > 
> > 
> > did you solve the problem? ive got the same... plugin doesnt work for me
> 
> I have the same problem

ASF change its bot config format, `AutoSteamSaleEvent` now merge in a option of `FarmingPerferences`, check if you enabled this feature,

ASFEnhance now supported "ENABLEAUTOSTEAMSALEEVENT` command, use this command to enable this feature for speect bots

**Reply by chr233** [reply]
> > yes, after that it takes some time to start, steam is a bit laggy but it worked for me
> 
> did you solve the problem? ive got the same... plugin doesnt work for me

ASF change its bot config format, `AutoSteamSaleEvent` now merge in a option of `FarmingPerferences`, check if you enabled this feature,

ASFEnhance now supported "ENABLEAUTOSTEAMSALEEVENT` command, use this command to enable this feature for speect bots

**Reply by Vadson159** [reply]
If the command ENABLEAUTOSTEAMSALEEVENT does not work for you, just add to each bots config file the following line:

"FarmingPreferences": 128,

That will make command for discovery queue work again

**Reply by chr233** [reply]
> If the command ENABLEAUTOSTEAMSALEEVENT does not work for you, just add to each bots config file the following line:
> 
> "FarmingPreferences": 128,
> 
> That will make command for discovery queue work again

yep, it does the same thing

**Reply by psy0ch** [reply]
I assumed that !ENABLEAUTOSTEAMSALEEVENT does also include the !claimitem, but it seems like it doesnt. 
Is there a way to let !claimitem automatically run?

**Reply by chr233** [reply]
> I assumed that !ENABLEAUTOSTEAMSALEEVENT does also include the !claimitem, but it seems like it doesnt. Is there a way to let !claimitem automatically run?

"AutoSteamSaleEvent" is a ASF fearute, and `ENABLEAUTOSTEAMSALEEVENT` is a shortcut to enable this feature.

`claimitem` is a ASFEnhance feature, and if you want to automatically claim, you need to set your bot names in `AutoClaimItemBotNames` property (as the given image)

![image](https://github.com/chr233/ASFEnhance/assets/13798576/57e6f56d-8698-4c73-b4cb-3fc017172950)
![image](https://github.com/chr233/ASFEnhance/assets/13798576/293fa601-d0cc-4bf9-ae5b-5c8c641692fe)
